tck/features/EqualsAcceptance.feature

```diff
@@ -138,7 +138,43 @@ Feature: EqualsAcceptance
       | lhs           | rhs           | result |
       | [1, 2]        | [1]           | false  |
       | [null]        | [1]           | null   |
-      | ['a']         | [1]           | null   |
+      | ['a']         | [1]           | false  |
       | [[1]]         | [[1], [null]] | false  |
       | [[1], [2]]    | [[1], [null]] | null   |
       | [[1], [2, 3]] | [[1], [null]] | false  |
+
+  Scenario Outline: Comparing NaN
+    Given any graph
+    When executing query:
+      """
+      RETURN <lhs> = <rhs> AS isEqual, <lhs> <> <rhs> AS isNotEqual
+      """
+    Then the result should be:
+      | isEqual   | isNotEqual   |
+      | <isEqual> | <isNotEqual> |
+    And no side effects
+
+    Examples:
+      | lhs       | rhs       | isEqual | isNotEqual |
+      | 0.0 / 0.0 | 1         | false   | true       |
+      | 0.0 / 0.0 | 1.0       | false   | true       |
+      | 0.0 / 0.0 | 0.0 / 0.0 | false   | true       |
+      | 0.0 / 0.0 | 'a'       | false   | true       |
```

Comment:
You could simplify this `Outline` and just inline (heh) the `isEqual` and `isNotEqual` as constants in the result expectation, as these values do not change per row in the `Examples` table.